src/full/Agda/TypeChecking/Positivity.hs

```diff
@@ -553,12 +558,12 @@ computeOccurrences' q = inConcreteOrAbstractMode q $ \ def -> do
               -- In any case, if @t@ is not showing itself as the data type, we need to
               -- do something conservative.  We will just collect *all* occurrences
               -- and flip their sign (variance) using 'LeftOfArrow'.
-              let fallback = OccursAs LeftOfArrow <$> getOccurrences (vars $ size tel) t -- NB::Defined but not used
+              let fallback = OccursAs LeftOfArrow <$> getOccurrences varsTel t -- NB::Defined but not used
```

Comment:
Is there any reason to keep this definition rather than removing it?

Reply:
I don't know why that code is still there. Perhaps @andreasabel does.

Reply:
You can delete it, and also its reference in the comment of one of the impossible cases below.  But please keep the link to issue 4447 in the comment.

Reply:
Will do as I bump the `cubical` submodule, once agda/cubical#1006 passes.